Header header
int8 id
float64[3] pos
float64[3] vel
float64[3] posGoal
float64[3] velGoal
